test/cli/samples/custom-frame/rollup.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	input: 'main.js',
+	output: {
+		format: 'cjs'
+	},
+	plugins: [
+		{
+			transform() {
+				const err = new Error('My error.');
+				err.loc = {
+					line: 1,
+					column: 5,
+				};
+				err.frame = 'custom code frame';
```

Comment:
It puzzled me a little why your test was green even without your changes to the code and it seems this test does not even trigger your code changes as no code position is provided as the second argument. This does not mean that I would change this test—it definitely tests something important—but that there should be a second test where `this.error` is called with a second argument, either a `{line, column}` object or a plain number (or just call it twice :) you might also just put all three invocations into one test).